index.js

```diff
@@ -203,7 +207,11 @@ Object.getOwnPropertyNames(Promise.prototype).forEach(method => {
 })
 
 function isInjection (obj) {
-  return (obj instanceof Request || obj instanceof Response)
+  return (
+    obj instanceof Request ||
+    obj instanceof Response ||
+    obj?.constructor?.name === '_CustomLMRRequest'
```

Comment:
Optional Chain is not supported by `node@10`
```suggestion
    (obj && obj.constructor && obj.constructor.name === '_CustomLMRRequest')
```